### PR TITLE
Update cses-1196.mdx

### PR DESCRIPTION
**Pull request by anuragdw710**
Edited code of gold/cses-1196

_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/gold/cses-1196.mdx [hunk 1]
```diff
@@ -13,305 +13,50 @@ iterate through the adjacency list of each vertex at most $k$ times.
...
-		ans.pb(bes[n].top());
+	while (bes[n].size()) {
+		ans.push_back(bes[n].top());
 		bes[n].pop();
 	}
-	reverse(all(ans));
+	reverse(ans.begin(), ans.end());
 	for (auto a : ans) cout << a << " ";
+
+	return 0;
```
Comment (SansPapyrus683) [inline solutions/gold/cses-1196.mdx:60]:
```suggestion
```

**Review by SansPapyrus683 — APPROVED** [review]
always appreciate detemplating
gonna need an screenshot of the code being accepted tho

**Comment by SansPapyrus683** [comment]
i need a screenshot of it passing all the test cases

**Comment by anuragdw710** [comment]
I have updated the screenshot.

File: solutions/gold/cses-1196.mdx [hunk 2]
```diff
@@ -13,305 +13,48 @@ iterate through the adjacency list of each vertex at most $k$ times.
...
+vector<pair<int, int>> adj[MX];
 priority_queue<pair<ll, int>, vector<pair<ll, int>>, greater<pair<ll, int>>> pq;
 
 int main() {
 	ios_base::sync_with_stdio(0);
 	cin.tie(0);
+
 	cin >> n >> m >> k;
-	F0R(i, m) {
+	for (ll i = 0; i < m; i++) {
```
Comment (ryanchou-dev) [inline solutions/gold/cses-1196.mdx]: this doesn't have to be a long long right?

**Review by ryanchou-dev — APPROVED** [review]
awesome! thank you.

**Pull request merged**